doc/nrf/includes/thread_configure_network.txt

```diff
@@ -0,0 +1,10 @@
+Configure the required Thread network parameters with the ``ot channel``, ``ot panid`` and ``ot masterkey`` commands:
+
+.. code-block:: console
+
+   uart:~$ ot channel 11
```

Comment:
This is redundant, added it as we already have panid and masterkey. 
These three usually are configured together.